src/qt/trafficgraphwidget.cpp

```diff
@@ -126,12 +177,17 @@ void TrafficGraphWidget::updateRates()
 {
     if(!clientModel) return;
 
+    static int64_t nTime = GetTimeMillis();
+    static int64_t nLastTime = nTime - timer->interval();
+    int nRealInterval = nTime - nLastTime;
```

Comment:
This code also helps to fix the "spike" issue that can happen when the real interval is different to the timer->interval() value.